sdk/lib/crt/msvcrtex.cmake

```diff
@@ -93,12 +93,9 @@ endif()
 set_source_files_properties(${MSVCRTEX_ASM_SOURCE} PROPERTIES COMPILE_DEFINITIONS "_DLL;_MSVCRTEX_")
 add_asm_files(msvcrtex_asm ${MSVCRTEX_ASM_SOURCE})
 
-add_library(msvcrtex ${MSVCRTEX_SOURCE} ${msvcrtex_asm})
+add_library(msvcrtex OBJECT ${MSVCRTEX_SOURCE} ${msvcrtex_asm})
```

Comment:
Making msvcrtex an object library caused problems in gdi32/user32 for me (duplicate symbols, since all the objects are now linked in unconditionally). I just reverted to a static library to fix that.